datatypes/src/operations/image/rgba_transmutable.rs

```diff
@@ -0,0 +1,161 @@
+use crate::operations::image::RgbaColor;
+
+/// This trait allows using raw bytes as RGBA colors
+///
+/// # Examples
+///
+/// ```
+/// use geoengine_datatypes::operations::image::{RgbaColor, RgbaTransmutable};
+///
+/// assert_eq!(
+///     0x01234567_u32.transmute_to_rgba(),
+///     RgbaColor::new(0x01, 0x23, 0x45, 0x67)
+/// );
+/// ```
+pub trait RgbaTransmutable {
+    fn transmute_to_f64(self) -> f64;
```

Comment:
you can remove this and implement transmute_to_rgba directly

Reply:
68fcf96